tests/Set/DoctrineORM213Set/Fixture/replace.php.inc

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Rector\Doctrine\Tests\Set\DoctrineORM213Set\Fixture;
+
+use Doctrine\Common\EventSubscriber;
+use Doctrine\ORM\Event\OnFlushEventArgs;
```

Comment:
The test runs on `use Doctrine\ORM\Event\OnFlushEventArgs` but the rule goes on `Doctrine\ORM\Event\PreFlushEventArgs`.

Looks there are some more Classes needed to the ruleset which has `getEntityManager` deprecated.